.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
You should include `node_modules` folder here to avoid upload all the dependencies. Please look for a `.gitignore` file example for Node express projects on Google